app/models/pet.rb

```diff
@@ -33,6 +34,7 @@ class Pet < ApplicationRecord
   has_one :match, dependent: :destroy
   has_many_attached :images
   enum species: ["dog", "cat"]
+  enum placement_type: ["adoptable", "fosterable", "adoptable and fosterable"]
 
   validates :name, presence: true
```

Comment:
I think it would be a required field for the app to function properly, so we should validate presence?